code/modules/tgui_panel/stat.dm

```diff
@@ -79,6 +90,7 @@
 	if(!is_ready())
 		return
 	window.send_message("stat/deadPopup", list())
+	addtimer(CALLBACK(src, .proc/clear_dead_poup), 3 MINUTES)
```

Comment:
Is this needed? Timers are somewhat expensive so we should try to avoid them where we don't need them

Reply:
Unless you have other idea how to make it clear by itself it is needed.
If it's that big of an issue I can remove it